scania/corporate-ui-site
# Icons page/select download icons

**Describe pull-request**  
 - When select all, check all boxes
- Add function to save selected items
- Added jszip helper to download icons as zip
- added check and uncheck icons function
- added category as input in icon-list component
- When download ongoing, add state in button
- Added color fallback for IE

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: #104 

**How to test**  
1. Check functionality to select & deselect icons
2. try download icons in zip

**Screenshots**  
![image](https://user-images.githubusercontent.com/1199101/95063355-86535480-06fe-11eb-8ca4-ef8be0d60873.png)
![image](https://user-images.githubusercontent.com/1199101/95063493-ae42b800-06fe-11eb-8085-e9fc66c8a7e4.png)


**Additional context**  
_Add any other context about the pull-request here._
